### PR TITLE
Add textmate-unstable

### PR DESCRIPTION
**Pull request by core-code**
As discussed here:

https://github.com/Homebrew/homebrew-cask/pull/67190

### CODE REVIEW

File: Casks/textmate-latest.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask 'textmate-latest' do
```
Comment (vitorgalvao) [inline Casks/textmate-latest.rb]: `latest` is ambiguous for the mais repo. Would `unstable` make sense?

Comment (core-code) [reply Casks/textmate-latest.rb]: you are right, that makes sense. i can rename once i am in front of a computer

Comment (reitermarkus) [reply Casks/textmate-latest.rb]: How about `textmate-preview`? The preference `.plist` was called `com.macromates.TextMate.preview` at some point in the past at least.

Comment (core-code) [reply Casks/textmate-latest.rb]: i don't have an opinion here. whatever you decide, let me know and i'll rename accordingly.

Comment (vitorgalvao) [reply Casks/textmate-latest.rb]: I’m fine with `preview`.

**Pull request merged**